edge-hub/src/Microsoft.Azure.Devices.Routing.Core/util/SystemTime.cs

```diff
@@ -1,12 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-namespace Microsoft.Azure.Devices.Routing.Core.Util
-{
-    using System;
```

Comment:
Are these just not used?

Reply:
There were two identical implementation, I kept the one from Devices.Edge.Util. Same for the other classes - sometimes the one in Devices.Edge.Util was improved (but had the functionality of the one from Routing.Core.Util) and sometimes just simple copies. Also the test classes were duplicated

Reply:
Ok cool!